semgrep-core/src/core/Range.mli

```diff
@@ -19,6 +19,8 @@ val ( $<=$ ) : t -> t -> bool
 (* disjoint *)
 val ( $<>$ ) : t -> t -> bool
 
+val range_of_line_spec : string -> Common.filename -> t
```

Comment:
It would be great to explain what kind of string format is expected as input. Same for the old function below (`range_of_linecol_spec`).